hack/functests.sh

```diff
@@ -68,9 +68,19 @@ if [ "$KUBEVIRT_E2E_PARALLEL" == "true" ]; then
         serial_test_args="${serial_test_args} --focus=\\[Serial\\]"
     fi
 
+    return_value=0
+    set +e
     functest --nodes=${KUBEVIRT_E2E_PARALLEL_NODES} ${parallel_test_args} ${FUNC_TEST_ARGS}
+    return_value="$?"
+    set -e
+    if [ "$return_value" -ne 0 ] && ! [ "$KUBEVIRT_E2E_RUN_ALL_SUITES" == "true" ]; then
+        exit "$return_value"
+    fi
     extra_args="-junit-output ${ARTIFACTS}/partial.junit.functest.xml"
     functest ${serial_test_args} ${FUNC_TEST_ARGS}
+    if [ "$return_value" -ne 0 ]; then
```

Comment:
Nit, at this point we could just `exit "$return_value"`, no need for if as I understand it, but maybe it would be less clear, wdyt?

Reply:
You mean like this?
```suggestion
    [ "$return_value" -ne 0 ] && exit "$return_value"
```

Reply:
No, I mean that, given that this is the last statement in the file, no matter if `$return_value` is 0 or not we are going to exit with its value. With the `if`, when `$return_value` is not 0, we exit with its value, when it is zero we exit with its value too (0). Without the `if` (plain `exit "$return_value"` instead if the `if`) we exit with "$return_value", same as with the `if`.

Reply:
Done

Reply:
Ah, ok, that makes sense :)